framework/docs/source/conf.py

```diff
@@ -67,7 +67,10 @@
     [
         tag.name
         for tag in repo.tags
-        if int(tag.name[1]) > 0 and int(tag.name.split(".")[1]) >= 8
+        if not tag.name.startswith("intelligence/")
+        and tag.name[0] == "v"
```

Comment:
what is this new `"v"` check for?

Reply:
I think its to ensure that the version tag starts with `v` (as in `flowerintelligence/v0.1.0`)

Reply:
It is just to make sure that we have a tag of the format "v%d.%d.%d", because we later take the second element of the string array
This is probably "overkill", I can remove it if you prefer

Reply:
Let's keep it

Reply:
let's keep it ~